### DISCUSSION TITLE
what's the license ?

### DISCUSSION THREAD

**Discussion by jsbimra** [speechly/speechly#154] [General]
Speechly can be used for commercial projects or its free for development projects ?
What is license condition of general use or in any project ?

**Comment by aukkonen** [comment]
@jsbimra 
Hello!

As written on our [pricing page](https://www.speechly.com/pricing), using the Speechly API is currently free for small-scale, personal, non-profit work, such as student projects. For commercial projects we offer the Professional package, as well as a number of enterprise tiers with extended features.

For institutional non-profit projects, please be in touch with us by filling [this form](https://www.speechly.com/contact).

**Comment by jsbimra** [comment]
okay noted. Thanks
